legal-api/migrations/versions/004f7558063f_second.py

```diff
@@ -1069,6 +1064,39 @@ def downgrade():
         batch_op.drop_column('last_ledger_id')
         batch_op.drop_column('last_modified')
 
+    with op.batch_alter_table('alternate_names', schema=None) as batch_op:
+        batch_op.drop_constraint(None, type_='foreignkey')
```

Comment:
Just a learning question, why is this line repeated 6 times?

Reply:
It is just a way of dropping the 6 FK constraints created for `alternate_names` table.  I believe you can delete the FK by the FK name explicitly as well.

Reply:
Noted, thanks!